checks/pagerduty.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: 'PagerDuty Integration'
+backlink_href: /docs
+backlink_title: 'Documentation'
+---
+
+## PagerDuty + pganalyze Integration Benefits
+
+ * Notify on-call responders based on alerts sent from pganalyze
+ * Link PagerDuty incidents to issue details in pganalyze
+ * Automatically acknowledge PagerDuty incidents when issues are acknowledged in pganalyze
+ * Automatically resolve PagerDuty incidents when pganalyze detects an issue has been resolved
+
+## How it Works
+
+In order to ensure your database is running smoothly, pganalyze runs a number of checks every time
+it receives statistics snapshots about your database from the [Collector](/docs/collector).
+These checks can create issues in pganalyze, which you can view by navigating to the **Alerts & Check-Up**
+page in the left menu in pganalyze. Once the underlying problem has been resolved, pganalyze will
+automatically resolve the issue.
+
+Integration with PagerDuty can trigger incidents on a PagerDuty service when new critical issues are
+created (or when existing issues escalate to critical severity). Once a pganalyze issue has been resolved,
+the associated PagerDuty incident will be automatically resolved as well.
+
+
+## Requirements
+
+In order to set up the initial integration with PagerDuty, you will need [modify permissions](/docs/permissions)
+on the pganalyze organization. In order to configure alerts for a specific server, you will need **modify**
+permissions on that server.
+
+
+## Support
+
+If you need help with this integration, please contact [support@pganalyze.com](mailto:support@pganalyze.com).
+
+
+## Integration Walkthrough
+
+Follow these instructions for your selected organization in pganalyze.
+
+
+### Initial setup
+
+Before configuring alerts, you will need to connect your pganalyze organization to PagerDuty:
+
+1. In the top menu, click the **x** in the **Server** dropdown if a server is selected to clear the active server
+2. In the left menu, click the **Settings** item
+3. On the organization settings page, click the **Integrations** tab
+4. In the panel labeled **PagerDuty**, click the **Connect to PagerDuty** button
+5. Continue to authenticate with PagerDuty and select which services you would like to connect to pganalyze
+   * services you select will later be available to configure for alerting; no alerting is enabled by default
```

Comment:
I think we can uppercase `services` here, since it's at the start of a sentence.

Reply:
Sure--I thought as a sub-item this could work lowercase, but you're right, when rendered it reads better capitalized.